### DISCUSSION TITLE
Add CLI command to show the list of current collections

### DISCUSSION THREAD

**Discussion by islamaliev** [sourcenetwork/defradb#1284] [Feature request]
### Is the request related to a problem?

It might be hard for a user who is not familiar with the database structure to get started and make changes.
At the moment it's not possible to simply get a list of collections in the database using CLI.
Yes, one can write an introspection query but it implies some effort.

### The feature you would like

Would be nice to have something like `defradb client collection show` that would just output the list of collections in the database. Like:
```
authors
books
publishers
```

**Comment by orpheuslummis** [comment]
Interesting idea

Further basic commands that could maybe exist:

- `collection create <collection_name>`: Create a new collection with the specified name.
- `collection delete <collection_name>`: Delete an existing collection by its name.
- `collection rename <old_collection_name> <new_collection_name>`: Rename an existing collection.
- `collection describe <collection_name>`: Show detailed information about a collection, such as fields, indexes, and schema.

Also `collection list` is a potential alternative to `collection show`. (in mysql `show` lists tables, but in other sql databases that i know of `show` displays values of parameters.)
